docs/spark-standalone.md

```diff
@@ -278,6 +278,16 @@ SPARK_MASTER_OPTS supports the following system properties:
   </td>
   <td>1.1.0</td>
 </tr>
+<tr>
+  <td><code>spark.deploy.spreadOutDrivers</code></td>
+  <td>true</td>
+  <td>
+    Whether the standalone cluster manager should spread drivers out across nodes or try
+    to consolidate them onto as few nodes as possible. Spreading out is usually better for
+    data locality in HDFS, but consolidating is more efficient for compute-intensive workloads. <br/>
```

Comment:
Nit: I noticed that this trailing br tag is derived from existing documentation, but it appears to have no contribution to the documentation layout.